packages/transactions/src/coders/receipt.ts

```diff
@@ -774,6 +774,18 @@ export const getMintedAssetId = (contractId: string, subId: string): string => {
   return sha256(concat([contractIdBytes, subIdBytes]));
 };
 
+type AssetIdInput = {
+  bits: string;
+};
+
+export const createAssetId = (contractId: string, subId: string): AssetIdInput => {
+  const bits = getMintedAssetId(contractId, subId);
+  const assetId: AssetIdInput = {
+    bits,
+  };
+  return assetId;
+};
```

Comment:
```suggestion
export const createAssetId = (contractId: string, subId: string): AssetId => ({
  bits: getMintedAssetId(contractId, subId),
});
```

Simplified and please note I've used the `AssetId` from `fuel-ts/interfaces`, you will need to import that.